MdeModulePkg/Universal/Variable/RuntimeDxe/VariableSmmRuntimeDxe.c

```diff
@@ -1898,7 +1898,9 @@ VariableSmmRuntimeInitialize (
   //
   gBS->CreateEventEx (
          EVT_NOTIFY_SIGNAL,
-         TPL_NOTIFY,
+         // MU_CHANGE - Only transition Variable Services to Runtime AFTER all Notify and Callback signals
+         //             so that volatile variables are writeable in callbacks.
+         TPL_CALLBACK-1,

```

Comment:
Is it legit to do -1 here? Should we be doing TPL_APPLICATION instead? Or is it intended to have levels between the defined levels?

Reply:
This is ugly but these in-between TPL levels have had some serious effects on order execution.  I'm not saying we shouldn't change this to something like TPL_APPLICATION but it should be tested.

Reply:
This is intentional. You can't create a notify event at `TPL_APPLICATION`.